lingua_franca/internal.py

```diff
@@ -37,6 +37,10 @@
 # commit 35efd0661a178e82f6745ad17e10e607c0d83472 for the "proper" state
 # of affairs, raising the errors below instead of deprecation warnings
 
+# For ease of development, in order to be able to type `if not lang:`
```

Comment:
can you explain better? if ```lang is None```,  then ```if lang:``` is falsish

you can do ```if not lang:``` freely in all cases

```
lang = None
if not lang:
    print("not lang None")
lang = ""
if not lang:
    print("not lang empty")
lang = False
if not lang:
    print("not lang Bool")
```

Reply:
Well, this change is necessary because the deprecation warnings currently rely on `lang` being `None` only when the parameter is actually passed: `nice_duration(60, lang=None)`

We had been using `lang=None` as the default param for all these functions, and I'd like to continue doing so, but the interaction between the two causes the DeprecationWarnings to come up *by default*, when no language is passed.

Hence, right now, and until the None-lang deprecation happens, this module draws a distinction between `lang is None` and `lang == ''`

Reply:
In fact, this makes most of the test suite warnings go away.

Reply:
i get that, but you said 

	in order to be able to type `if not lang:`

which i maybe misunderstood as you saying "if lang == "" this will fail"

Reply:
what i am saying is that i disagree with this comment, i don't see a need to make the default None again at a later date